login.php

```diff
@@ -49,11 +49,19 @@
 	</head>
 	
 	<body>
+		<!-- This class is to split the content in half and places it to the left-->
+		<div class="section1">
```

Comment:
Trialled different positioning for the login where the logo would be enlarged and placed on the left and the actual login would be to the right